File: curso_c_cpp/c/01_introduccion/ejemplos/varias_lineas.c
/*
 * Ejemplo 2 - Varios printf y secuencias de escape (\n salto de linea, \t tabulacion).
 */
#include <stdio.h>

int main(void) {
    printf("Aprendiendo C\n");
    printf("Linea 1\nLinea 2\n");     /* dos lineas en un solo printf */
    printf("Nombre:\tEdad:\n");       /* \t separa con una tabulacion */
    printf("Ana\t25\n");
    printf("Se imprime una comilla: \" y una barra: \\\n");
    return 0;
}

File: curso_java/01_introduccion/ejemplos/VariasLineas.java
/*
 * println vs print, y secuencias de escape (\n salto de linea, \t tabulacion).
 */
public class VariasLineas {
    public static void main(String[] args) {
        System.out.println("Aprendiendo Java");
        System.out.println("Linea 1");
        System.out.println("Linea 2");

        System.out.print("Sin ");        // print NO salta de linea
        System.out.print("salto\n");     // \n hace el salto manualmente

        System.out.println("Nombre:\tEdad:");   // \t = tabulacion
        System.out.println("Ana\t25");
        System.out.println("Comilla \" y barra \\");
    }
}
